src/core_plugins/kibana/public/context/query_parameters/__tests__/action_set_query_parameters.js

```diff
@@ -0,0 +1,50 @@
+import expect from 'expect.js';
+
+import { createStateStub } from './_utils';
+import { QueryParameterActionsProvider } from '../actions';
+
+
+describe('context app', function () {
+  describe('action setQueryParameters', function () {
+    it('should update the queryParameters with certain values from the given object', function () {
```

Comment:
Similar to below, "certain values" doesn't really mean anything to me. Would something like "should update the queryParameters with valid properties from the given object" sound ok to you?